docs/source/sample_datasets.rst

```diff
@@ -30,22 +30,48 @@ Creating random data
 
 Random data can be generated in the following way. It should be kept in mind
 that the data generated in this way can only be used for simple testing as it
-has no physical significance.
+has no physical significance. You can then load the data through the
+:ref:`api documentation`.
+
+**Raw file:**
 
 .. code-block:: python
-      
+
+    # Create sample raw file
     import numpy as np
     real_data = np.random.randn(16, 16, 16, 16).astype("float32")
     real_data.tofile("/tmp/real_raw_file.raw")
 
-Now you can load the data through the :ref:`api documentation` in the following way:
+.. code-block:: python
+
+    # Load the file
+    from libertem.api import Context
+    if __name__ == '__main__':
+      ctx = Context()
+      ds = ctx.load("raw", path="/tmp/real_raw_file.raw", scan_size=(16, 16), dtype="float32", detector_size=(16, 16))
+
+**HDF5 file:**
 
 .. code-block:: python
-    
+
+    # Create sample HDF5 file
+    import h5py
+    import numpy as np
+    file = h5py.File('hdf5_sample.h5','w')
+    dataset = file.create_dataset("dataset",(16,16,16,16), h5py.h5t.STD_I32BE)
+    dataset = file['/dataset']
+    data = np.random.randn(16,16,16,16).astype("float32")
```

Comment:
Hmm, you are creating float data, but setting the `dtype` of the HDF5 dataset to `h5py.h5t.STD_I32BE` - that is, 32bit big-endian integers. I don't think this will work as intended. Instead of specifying the `dtype` explicitly, you can also provide the `data` argument instead, [see the reference docs](http://docs.h5py.org/en/stable/high/group.html#Group.create_dataset) and [the creating datasets](http://docs.h5py.org/en/stable/high/dataset.html#dataset-create) section for details.

Reply:
Sorry, thanks!